cloud/pkg/cloudhub/config/config.go

```diff
@@ -25,7 +25,7 @@ type Configure struct {
 func InitConfigure(hub *v1alpha1.CloudHub) {
 	once.Do(func() {
 		if len(hub.AdvertiseAddress) == 0 {
-			klog.Exit("AdvertiseAddress must be specified!")
+			panic("AdvertiseAddress must be specified!")
```

Comment:
`panic(errors.New("AdvertiseAddress must be specified!"))`
Plz input an error value to `panic(..)` func. If `panic(..)` input a string value, then `recover(..)` returns a string value

Reply:
Thanks, Fixed this